Makefile.am

```diff
@@ -170,6 +170,14 @@ libtesseract_la_LIBADD += libtesseract_avx512.la
 noinst_LTLIBRARIES += libtesseract_avx512.la
 endif
 
+if HAVE_AVX512VNNI
+libtesseract_avx512vnni_la_CXXFLAGS = -mavx512vnni -mavx512vl
```

Comment:
I am curious: what is the use of `-mavx512vl`? Should it be checked in `configure`, too?

Reply:
It does not compile without the second flag. It is needed becaIuse the code uses ymm registers (256 bit) instead of zmm registers (512 bit).

Yes, we can also add another check in configure.

Note that AFAIK all intel's CPUs that support AVX512VNNI also support AVXVL.

Reply:
Thanks for explaining. Maybe we don't need the additional check then.